.github/workflows/scylla.yaml

```diff
@@ -0,0 +1,46 @@
+name: Build Scylla
+
+on:
+  schedule:
+    # only at 5AM Saturday
+    - cron: '0 5 * * SAT'
+
+permissions: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  BUILD_TYPE: RelWithDebInfo
+  BUILD_DIR: build
+
+jobs:
+  build-scylla:
+    runs-on: ubuntu-latest
+    container: fedora:40
+    steps:
+      - run: |
+          sudo dnf -y install git
+      - uses: actions/checkout@v4
+        with:
+          repository: scylladb/scylladb
+          submodules: true
+      - name: Update seastar submodule
+        run: |
+          git config --global --add safe.directory '*'
+          git fetch origin master
+          git reset --hard origin/master
+      - run: |
+          sudo ./install-dependencies.sh
+      - name: Generate the building system
+        run: |
+          cmake                                         \
+            -DCMAKE_BUILD_TYPE=$BUILD_TYPE              \
+            -DCMAKE_C_COMPILER=clang                    \
+            -DCMAKE_CXX_COMPILER=clang++                \
+            -G Ninja                                    \
+            -B $BUILD_DIR                               \
+            -S .
+      - run: |
+          cmake --build $BUILD_DIR --target scylla
```

Comment:
other tagets depends on seastar too, like unit tests

Reply:
also, better cover more build modes.
release, dev, and debug to begin with

Reply:
sure. added.

Reply:
> sure. added.

I don't see other targets in [`cbc81b9` (#2284)](https://github.com/scylladb/seastar/pull/2284/commits/cbc81b922317daa7b1160f3266e127e18f617c2b)